content/docs/alerting/notifications.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Notification template reference
+sort_rank: 6
+---
+# Notification Template Reference
+
+Prometheus creates and sends alerts to the Alertmanager which then sends notifications out to different receivers based on their labels.
+A receiver can be one of many integrations including: Slack, PagerDuty, email, or a custom integration via the generic webhook interface.
+
+The notifications sent to receivers are constructed via templates. The Alertmanager comes with default templates but they can also be customized.
+To avoid confusion it's important to note that the Alertmanager templates differ from [templating in Prometheus](https://prometheus.io/docs/visualization/template_reference/).
+
+The Alertmanager's notification templates are based on the [Go templating](http://golang.org/pkg/text/template) system.
```

Comment:
You should mention that some fields are evaluated as text, other as HTML. This impacts escaping.